### ISSUE TITLE
[Android] landscapeRightMode  rotates the screen twice times

### ISSUE THREAD

**Issue by fantasy525** [bytepark/auto_orientation#8]
```
  auto_orientation: ^1.0.5
  flutter:1.7.8-hotfix4
```
1. invoke` await AutoOrientation.landscapeRightMode();`
2.screen rotates twice times.
I found that I comments out this line of code,it can work ok.
```
  static landscapeRightMode() async {
    // delete below line
    // SystemChrome.setPreferredOrientations([DeviceOrientation.landscapeRight]);
    try {
      await _channel.invokeMethod('setLandscapeRight');
    } on MissingPluginException catch (_) {
      return;
    }
  }
```

**Comment by sfaujour** [comment]
Thanks for the issue - which phone / os are you using?

**Comment by anjanjanj** [comment]
same issue here - OnePlus 7T

**Comment by ashemetov** [comment]
Any chance to fix? Samsung S21 has the same issue.

**Comment by KasunSKarunasekara** [comment]
Having the same issue on Nokia 6.1 plus? Will this be fixed?

**Comment by rahul-insight** [comment]
Having the same issue on iOS. Will it be fixed?

**Comment by fbatschi** [comment]
This should be fixed by 31ec48b6ea331fb802996aa29e73b01e93ddc611
will be published soon as new package
